website/src/workers/worker.js

```diff
@@ -1,4 +1,5 @@
 import init, * as aleo from "@aleohq/wasm";
+import { resolveImports } from "../utils.js";
```

Comment:
I think you might have forgotten to commit this file? Looks like the website build is failing + the dev server errors when you navigate to `/develop` for me.

Reply:
You're absolutely right! Pushed.